src/nodes/NamespaceDeclaration.php

```diff
@@ -18,8 +18,12 @@ public function getQualifiedNameAsString(): string {
     if ($name instanceof NameToken) {
       return $name->getText();
     }
+    
+    if ($name === null) {
+      return '';
```

Comment:
shouldn't this be : 
```hack
     return $this->getDescendantsOfType(NameToken::class)
       |> Vec\map($$, $t ==> $t->getText())
       |> Str\join($$, "\\");
```
? in that case, instead of write that code, you can do `$name = $this` without `return` which should do the same

Reply:
No - I think that would break with

```php
namespace {
  class A {}
  class B {}
}
```

Sadly I haven't time to add test cases at the moment, but will be more free in first few days of January.